opensearch/src/main/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactory.java

```diff
@@ -295,7 +295,7 @@ private static ExprValue createOpenSearchDateType(Content value, ExprType type)
         }
       } else {
         // custom format
-        return parseDateTimeString(value.stringValue(), dt);
+        return parseDateTimeString(value.objectValue().toString(), dt);
```

Comment:
could u elberate more? value is number value? why objectValue().toString()?

Reply:
The value is an instance of `ObjectContent` and isNumber() is true, `stringValue()` will return the string representation of the object, such as `ObjectContent@1234`. We should convert it to Java Object then use `toString()` here. Or `value.longValue().toString()`. Not sure the value here is a Long type, so `value.objectValue().toString()` would be best.